Musicplayer.css

```diff
@@ -82,47 +52,111 @@ nav a:hover{
 	transition:0.5s;
 }
 
-.content_1{
-	background-color: white;
+ul li ul{
+	padding: 0;
+	position: absolute;
+	top: 58px;
+	left: 130px;
+	width: 100px;
+	display: none;
+	visibility: hidden;
+	z-index: 1;
+}
+	
+ul li ul li{
+	background-color: purple;
+	display: block;
+	
+	
+}
+
+ul li ul li:hover{background: black}
+
+
+.logo img{
+	position:absolute;
+	margin-top:-20px;
+}
+
+.img_1{
 	grid-column-start: 1;
 	grid-column-end: span 4;
-	z-index: 2;	
-	height: 100px;
-	border: 2px solid red;
 }
 
+.img_1 img{
+	display:block;
+	margin-left:auto;
+	margin-right:auto;
+	height:400px;
+	width:100%;
+}
+
+.footer{
+	background-color: black;
+	grid-column-end: span 4;
+	padding-top:25px;
+	text-align: center;
+	color:white;
+	height:50px;
+}
+
+.content_1{
+	grid-column-start: 2;
+	grid-column-end: span 4;
+	background-color: white;
+	z-index: 2;
+	height:200px;
+	text-align:center;
+}
 
 .content_2{
 	grid-column-start: 1;
-	grid-column-end: span 1;
+	grid-column-end: 2;
 	background-color: white;
 	z-index: 2;
-	height:604px;
-	border: 2px solid red;
+	height:200px;
+}
+.content_2 img{
+	height:200px;
+	width:202px;
+	
 }
 
+.content_3{
```

Comment:
21/9: Because I break the table to two part, content_3 with hold the title and content_4 will hold the data(song, description). So I create a content_3 div box to hold the title(briefly setup for the grid, color vv)